.github/workflows/test-suite.yml

```diff
@@ -2,9 +2,15 @@ name: Test Suite
 
 on:
   pull_request:
+    paths:
+      - '.github/workflows/**'
```

Comment:
do we need it at all workflows changes, or just `test-suite.yml` here?

Reply:
This was done because a test workflow is required for pull requests so it was hanging with a status pending